packages/react-button/src/components/CompoundButton/CompoundButton.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```

Comment:
Are we were putting all the stories (including the top-level one) into the stories subfolder or does the container of stories sit with the component code?